samples/nrf9160/modem_shell/src/cloud/cloud_shell.c

```diff
@@ -18,14 +19,21 @@
 #include <net/nrf_cloud_pgps.h>
 #endif
 
+#define CLOUD_CMD_MAX_LENGTH 150
+
 BUILD_ASSERT(
 	IS_ENABLED(CONFIG_NRF_CLOUD_MQTT) &&
 	IS_ENABLED(CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD));
 
+extern struct k_work_q mosh_common_work_q;
+
 static struct k_work_delayable cloud_reconnect_work;
 #if defined(CONFIG_NRF_CLOUD_PGPS)
 static struct k_work notify_pgps_work;
 #endif
+static struct k_work cloud_cmd_work;
+
+static char shell_cmd[CLOUD_CMD_MAX_LENGTH];
```

Comment:
Looks like there should be one char more for the null termination, otherwise a 150 char command will write one byte past the array.
```suggestion
static char shell_cmd[CLOUD_CMD_MAX_LENGTH + 1];
```